src/components/panel/panel.spec.js

```diff
@@ -31,6 +31,41 @@ describe('$mdPanel', function() {
     attachToBody($rootEl);
   });
 
+  beforeEach(function() {
+    jasmine.addMatchers({
+      /**
+       * Asserts that two values are within a range of each other. This is
+       * used for testing relative positioning. If no range is set, defaults
+       * to 1.
+       *
+       * Example Use:
+       * expect(panelTop).toBeApproximately(buttonTop, 1);
+       */
+      toBeApproximately: function() {
```

Comment:
`toBeCloseTo` compares floats with a certain amount of decimal precision. I want to check that two numbers are within `x` of each other.

For instance, `expect(54.99).toBeCloseTo(55.01)` fails. With `toBeApproximately` this passes.